util/src/main/java/org/datacommons/util/ReconClient.java

```diff
@@ -0,0 +1,51 @@
+package org.datacommons.util;
+
+import static java.net.http.HttpClient.Version.HTTP_1_1;
+
+import com.google.protobuf.Message;
+import com.google.protobuf.util.JsonFormat;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse.BodyHandlers;
+import org.datacommons.proto.Recon.ResolveCoordinatesRequest;
+import org.datacommons.proto.Recon.ResolveCoordinatesResponse;
+
+/**
+ * Client to the DC resolution APIs.
+ *
+ * <p>Currently it only resolves coordinates.
```

Comment:
remove the <p>?

Reply:
The intellij formatter adds the paragraph tag. I don't like it either but have made peace with it.